docs/RunSettingsArgs.md

```diff
@@ -0,0 +1,54 @@
+  # RunSettings argument through commandline
+  
+  This argument will help you to specify runsettings settings through commandline arguments. For example, you can change the .NET Framework on which the tests will be run, the directory where test results are delivered during a test run.
+  
+  **Note:** Runsettings is a *.runsettings file which allow user to configure unit test. More info here :  https://msdn.microsoft.com/en-us/library/jj635153.aspx 
+  
+  ## Syntax
+  
+  `vstest.console.exe [Arguments] [Options] [[--] <args>...]]` or 
+  `dotnet test [options] <PROJECT> [[--] <additional arguments>...]]`
+  
+  `args` or `additional arguments` may be specified as name-value pair of the form `n=v`, where `n` is the argument name, and `v` is the argument value. All arguments after `--` will be treated as `additional arguments`. Use a space to separate multiple arguments.
```

Comment:
pl change args to additional arguments in vstest.console as well for consistency.